CITATION.cff

```diff
@@ -0,0 +1,26 @@
+cff-version: 1.2.0
+title: acquire-imaging
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Nathan
+    family-names: Clack
+    email: nclack@chanzuckerberg.com
+    affiliation: Chan-Zuckerberg Initiative Foundation
+    orcid: 'https://orcid.org/0000-0001-6236-9282'
+  - given-names: Alan
+    family-names: Liddell
+    email: aliddell@chanzuckerberg.com
+    affiliation: Chan-Zuckerberg Initiative Foundation
+  - given-names: Andy
+    family-names: Sweet
+    email: asweet@chanzuckerberg.com
```

Comment:
Prefer to use my personal email since it's already leaked all over the place anyway. Is that OK?

```suggestion
    email: andrewdsweet@gmail.com
```